src/main/java/io/codelex/oop/cars/Car.java

```diff
@@ -0,0 +1,75 @@
+package io.codelex.oop.cars;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private String name;
+    private String model;
+    private int price;
+    private int yearOfManufacture;
+    private ArrayList<Manufacturer> manufacturer = new ArrayList<>();
+    private EngineType engineType;
+
+
+    public Car(String name, String model, int price, int yearOfManufacture, List<Manufacturer> manufacturer, EngineType engineType) {
+        this.name = name;
+        this.model = model;
+        this.price = price;
+        this.yearOfManufacture = yearOfManufacture;
+        this.manufacturer.addAll(manufacturer);
+        this.engineType = engineType;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public ArrayList<Manufacturer> getManufacturer() {
+        return manufacturer;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public int getYearOfManufacture() {
+        return yearOfManufacture;
+    }
+
+    public EngineType getEngineType() {
+        return engineType;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Car car = (Car) o;
+        return price == car.price && yearOfManufacture == car.yearOfManufacture && Objects.equals(name, car.name) && Objects.equals(model, car.model) && Objects.equals(manufacturer, car.manufacturer) && engineType == car.engineType;
```

Comment:
if you have to scroll to the left, the line is too long 😄
https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html